.gitignore

```diff
@@ -24,5 +24,7 @@ _testmain.go
 *.prof
 .env
 
-coverage.out
+*.swp
```

Comment:
This should not be part of the gitignore, that should be handled on your own global gitignore.